annubes/tasks/task.py

```diff
@@ -16,24 +16,26 @@ class Task:
 
     Args:
         name: Name of the task.
-        session: Dictionary representing the ratio (values) of the different trials (keys) within the task. Trials with
-            a single modality must be represented by single characters, while trials with multiple modalities are
-            represented by the character combination of those trials. Note that values are read relative to each other,
-            such that e.g. `{"v": 0.25, "a": 0.75}` == `{"v": 1, "a": 3}` is True. Defaults to {"v": 0.5, "a": 0.5}.
-        stim_intensities: List of possible intensities of each stimulus.
+        session: Structure of the type of trials which can appear during a session.
+            It is given by a dictionary representing the ratio (values) of the different trials (keys) within the task.
+            Trials with a single modality (e.g., a visual trial) must be represented by single characters, while trials
+            with multiple modalities (e.g., an audiovisual trial) are represented by the character combination of those
+            trials. Note that values are read relative to each other, such that e.g. `{"v": 0.25, "a": 0.75}` ==
+            `{"v": 1, "a": 3}` is True.
+            Defaults to {"v": 0.5, "a": 0.5}.
+        stim_intensities: List of possible intensity values of each stimulus type.
             Defaults to [0.8, 0.9, 1].
         catch_prob: probability of catch trials in the session, between 0 and 1.
             Defaults to 0.5.
-        catch_intensity: Intensity value during a catch trial.
-            Defaults to 0.
-        shuffle_trials: If True (default), trial order will be randomized. If False, all trials of one modality are run
-            before any trial of the next modality starts, in the order defined in `session` followed by catch trials.
+        shuffle_trials: If True (default), trial order will be randomized. If False, all trials corresponding to one
+            modality (e.g. visual) are run before any trial of the next modality (e.g. auditory) starts, in the order
+            defined in `session` followed by catch trials.
         max_sequential: Maximum number of sequential trials of the same modality. Only used if shuffle is True.
             Defaults to None (no maximum).
-        n_outputs: Number of output signals that will be generated.
+        n_outputs: Number of output nodes in the network, signaling different behavioral choices.
             Defaults to 2.
-        output_intensities: List of possible intensities of the output signals. Currently only the smallest and largest
-            value of this list are used.
+        output_behavior: List of possible intensity values of the behavioral output. Currently only the smallest and
+            largest value of this list are used.
             Defaults to [0, 1].
```

Comment:
From what I understood, we decided to not use this after all, until we start setting up `Tasks` that actually will use this parameter. I think that should be done in a separate PR. Could you perhaps make an issue for that?